testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -1792,13 +1792,12 @@ void checkPeriodPredicate(Checker checker) {
     expr("cast('foo' as bar)")
         .ok("CAST('foo' AS `BAR`)");
   }
-  
+
   @Test void testParsingNonIsoCharacter() {
   	String sql = "select 'ק' ";
-		sql(sql).ok("SELECT u&'\\05e7'");
```

Comment:
why remove this test? The `u&` is valid standard SQL and is still valid Calcite SQL

Reply:
the CALCITE-6001 mentioned 
'This means that when converting a query like:
select 'ק' as result;
you will get the following the error: Failed to encode 'ק' in character set 'ISO-8859-1'.'

I justed it according to this behavior